Program/query_program_input.py

```diff
@@ -174,86 +164,85 @@
-
-    catalog_stars.write("tycho_{}.csv".format(run_count), format="csv", overwrite="True")
+    internal_id = ["Str_{}_{:08d}".format(const_abr[i],i+last_id+1) for i in range(len(catalog_stars['_RAJ2000']))]
+    catalog_stars.add_column(internal_id, name="Internal ID Number", index = 0)
+    
+    catalog_stars.rename_column('_RAJ2000','RAJ2000')
+    catalog_stars.rename_column('_DEJ2000','DEJ2000')
+    
+    catalog_stars.write("tycho_{}.csv".format(run_count),format="csv",overwrite = "True")
     last_id = last_id + len(catalog_stars['RAJ2000'])
 
     # Adding common names to the stars
     names_table = pd.read_csv('IAU-CSN.csv')
     names_table = pd.DataFrame(names_table)
-    ttable = pd.read_csv('tycho_{}.csv'.format(run_count), dtype={"Name": "string"})
+    ttable = pd.read_csv('tycho_{}.csv'.format(run_count),dtype = {"Name":"string"})
     ttable = pd.DataFrame(ttable)
-    names_table["HIP"] = np.array([x if x != '-' else 0 for x in names_table["HIP"]])
-    ttable.insert(1, "Name", np.zeros(len(ttable['TYC'])))
-    names_table["HIP"] = pd.to_numeric(names_table['HIP'], errors='coerce')
+    names_table["HIP"] = np.array([x if x!='-' else 0 for x in names_table["HIP"]])
+    ttable.insert(1,"Name",np.zeros(len(ttable['TYC'])))
+    names_table["HIP"] = pd.to_numeric(names_table['HIP'],errors='coerce')
     ttable['Name'] = ttable.HIP.map(names_table.set_index('HIP').Name)
     ttable.to_csv('tycho_{}.csv'.format(run_count))
-
+  
     # The cross cataloguing part
     ccat = pd.read_csv('CrossCatalog.csv')
     ccat = pd.DataFrame(ccat)
-    ccat["HIP"].fillna("0", inplace=True)
-    ttable = pd.read_csv('tycho_{}.csv'.format(run_count), dtype={"Name": "string"})
-    ttable = pd.DataFrame(ttable)
+    ccat["HIP"].fillna("0", inplace = True)
+    ttable = pd.read_csv('tycho_{}.csv'.format(run_count),dtype = {"Name":"string"})
+    ttable = pd.DataFrame(ttable) 
     ccat = ccat.loc[~ccat.HIP.duplicated(keep='first')]
-    ttable.insert(4, "Bayer", np.zeros(len(ttable['RAJ2000'])))
-    ttable['Bayer'] = ttable.HIP.map(ccat.set_index('HIP').BayerConst, na_action="ignore")
-    if run_count == 1:
-        ttable.to_csv('tycho-1.csv', index=False)
+    ttable.insert(4,"Bayer",np.zeros(len(ttable['RAJ2000'])))    
+    ttable['Bayer'] = ttable.HIP.map(ccat.set_index('HIP').BayerConst,na_action="ignore")
+    if run_count == 1:    
+        ttable.to_csv('tycho-1.csv',index=False) 
     else:
-        ttable.to_csv('tycho-1.csv', mode='a', header=False, index=False)
-
-    os.rename('tycho-1.csv', 'tycho-1_{}.csv'.format(maxV))  # final file will be tycho-1_{maximum_magnitude}.csv
-
+        ttable.to_csv('tycho-1.csv',mode='a', header=False,index=False)
+    
+#     os.rename('tycho-1.csv','tycho-1_{}.csv'.format(maxV)) # final file will be tycho-1_{maximum_magnitude}.csv
+    
     # below is where the +3 increment takes places
     minV = maxV
     maxV += 3
-
+    
     while True:
-        if maxV <= 12:
+        if maxV <=12:
```

Comment:
Why upto 12? If this is downloading all the way upto 12, then there will be redundant entries in the tycho-2 download.
Are we taking it up to 12 only so that there is an option not to download the big file at all?